config/deploy/prod.rb

```diff
@@ -6,4 +6,4 @@
 set :rails_env, 'production'
 
 set :sidekiq_role, :background
-set :sidekiq_processes, 10
+set :sidekiq_processes, 2
```

Comment:
I'm concerned that this change has been tuned somehow for production. Initially implemented here? https://github.com/sul-dlss/exhibits/commit/0d706d10855bb9f52a0f06dfbd26eb5a2a9bf214

Can we make the change in stage without doing it in prod yet?